Reachability/Reachability.swift

```diff
@@ -71,6 +71,8 @@ public class Reachability {
     public var whenReachable: NetworkReachable?
     public var whenUnreachable: NetworkUnreachable?
     public var reachableOnWWAN: Bool
+    
+    public var notificationCenter: NotificationCenter = NotificationCenter.default
```

Comment:
Personally I am using non-default notification centre as a bus for domain-specific events within my app. I believe it is just easier to track those events in the separate one.
Those are just 2 lines and still gives some additional flexibility.